src/openms/include/OpenMS/METADATA/ID/ParentGroup.h

```diff
@@ -44,35 +44,42 @@ namespace OpenMS
 {
   namespace IdentificationDataInternal
   {
-    /** @brief: Group of ambiguously identified parent molecules (e.g. protein group)
+    /** @brief: Group of ambiguously identified parent sequences (e.g. protein group)
     */
     // @TODO: derive from MetaInfoInterface?
-    struct ParentMoleculeGroup
+    struct ParentGroup
     {
       std::map<ScoreTypeRef, double> scores;
       // @TODO: does this need a "leader" or some such?
-      std::set<ParentMoleculeRef> parent_molecule_refs;
+      std::set<ParentSequenceRef> parent_refs;
```

Comment:
I think we have some ordering in our code. Is the operator< stable (e.g., based on the sequence) or some address?

Reply:
Maybe also a question for @jpfeuffer when he is back from vacation

Reply:
All "...Ref" types are ordered by address, so not stable. See `IteratorWrapper::operator<` (https://github.com/hendrikweisser/OpenMS/blob/98656f05c5be92a6be822e65f6c6f450f37e4a83/src/openms/include/OpenMS/METADATA/ID/MetaData.h#L51-L55).

If we're revisiting the design of the references in the context of pyOpenMS, we could rethink this decision.